pkg/cli/image/imagesource/s3.go

```diff
@@ -283,7 +283,7 @@ func (s *s3ManifestService) Put(ctx context.Context, manifest distribution.Manif
 		if _, err := s.r.s3.CopyObject(s.r.ctx, &s3.CopyObjectInput{
 			Bucket:      aws.String(s.r.bucket),
 			ContentType: aws.String(mediaType),
-			CopySource:  aws.String(path.Join(s.r.bucket, blob)),
+			CopySource:  aws.String(s.r.bucket + "/" + url.QueryEscape(blob)),
```

Comment:
Was there any reason to remove `path.Join` ?

Reply:
Do you want me still using path.Join?

Reply:
Yeah, it would be better to keep it.

Reply:
Updated.